webapp/backend/pkg/web/server.go

```diff
@@ -27,28 +27,33 @@ func (ae *AppEngine) Setup(logger logrus.FieldLogger) *gin.Engine {
 	r.Use(middleware.ConfigMiddleware(ae.Config))
 	r.Use(gin.Recovery())
 
-	api := r.Group("/api")
+	basePath := ae.Config.GetString("web.src.backend.basepath")
```

Comment:
can you explicitly set a default value (`""`) in [Config class](https://github.com/AnalogJ/scrutiny/blob/master/webapp/backend/pkg/config/config.go)

Also, this should be added to the [`example.scruiny.yaml`](https://github.com/AnalogJ/scrutiny/blob/master/webapp/backend/pkg/config/config.go)